### PR TITLE
CLOUDP-133816: Rename atlascli to mongodb-atlas

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes

Rename atlascli to mongodb-atlas

_Jira ticket:_ CLOUDP-133816

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: build/package/chocolatey/atlascli.nuspec [hunk 1]
```diff
@@ -2,12 +2,15 @@
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
-    <id>atlascli</id>
+    <id>mongodb-atlas</id>
```
Comment (fmenezes) [inline build/package/chocolatey/atlascli.nuspec:5]: this is the main change, all other changes were needed given we've had some instability on nupkg generation

File: tools/chocolateypkg/main.go [hunk 2]
```diff
@@ -97,29 +99,29 @@ func replaceNuspec(dir, version string) error {
 	return nil
 }
 
-func generateSha256(f *os.File) (string, error) {
+func generateSha256(url string) (string, error) {
+	r, err := http.NewRequestWithContext(context.Background(), http.MethodGet, url, http.NoBody)
```
Comment (fmenezes) [inline tools/chocolateypkg/main.go:103]: this is a super important change here we were generating the checksum from the .msi currently generated at the disk, here I'm changing to download package and generate checksum from it.

**Review by gssbzn — APPROVED** [review]
LGTM

**Review by andreaangiolillo — APPROVED** [review]
code LGTM

File: tools/chocolateypkg/main.go (hunk 2 above)
Comment (colm-quinn) [reply tools/chocolateypkg/main.go:103]: Nice! Was that the reason for the flakiness you spotted?

**Review by colm-quinn — APPROVED** [review]
LGTM!

**Pull request merged**